TransferMethod/Sources/SelectTransferMethodTypeController.swift

```diff
@@ -51,6 +51,11 @@ final class SelectTransferMethodTypeController: UITableViewController {
         presenter.loadTransferMethodKeys(forceUpdate)
     }
 
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        presenter.trackUILoadImpression()
```

Comment:
won't this lead to `trackUILoadImpression()` being called twice?

Reply:
also, when `viewDidAppear` calls `presenter.trackUILoadImpression()` then `loadedSuccessfully` is already true and so this second call translates into second trackImpression being called

Reply:
As discussed `presenter.trackUILoadImpression()` is being called twice after the first load, I think `loadedSuccessfully` needs to be reset back to `false` somewhere/sometime

Reply:
fixed

Reply:
fixed